.github/workflows/deploy-on-kind.yaml

```diff
@@ -0,0 +1,60 @@
+name: Testing
+on: push
```

Comment:
Does this need to only run if paths in the Jenkins helm chart have run?

i.e we don’t have other charts here atm, but there was plans to add the Jenkins operator at one point

Reply:
updated to only run on changes to `charts/jenkins/**`